src/main/java/net/masterthought/cucumber/json/Result.java

```diff
@@ -28,4 +28,8 @@ public String getFormatedDuration() {
     public String getErrorMessage() {
         return errorMessage;
     }
+
+    public final String getErrorMessageTitle() {
+        return errorMessage.split("[\\r\\n]+")[0];
```

Comment:
Based on https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html
\s should be enough